schema/EscSensorRecordExample.json

```diff
@@ -0,0 +1,1453 @@
+{
+  "id": "esc_sensor/admin_0/esc_sensor_0",
+  "installationParam": {
+    "latitude": 38.8,
+    "longitude": -100.25,
+    "height": 9.3,
```

Comment:
Missing the azimuth information
```
"antennaAzimuth": 0,
"antennaDowntilt": 0,
```

Reply:
thank you, I added them to the example, but for the schema they are optional, so do you think we should put them as required?

Reply:
From system point of view: the downtilt is definitely optional.
But the azimuth is mandatory otherwise ESC antenna gain can not be calculated.

This is under the assumption that a bug fix in the SAS-SAS spec is officially corrected, whereby
the pattern angles were defined against true north, instead of antenna boresight. Normally this
fix in the spec should be acted. 
So I guess that azimuth is "necessary". 

+ @kate-harrison :  do we need to fix the schema to mark them as required ? Does it affect the SAS/SAS spec, and do we need to have both in line ?

Reply:
Note: after 2nd thoughts, in fact it is perhaps not mandatory, if we use 0 degrees when not specified. In that case, we fall back on the mode where the antenna pattern is effectively defined according to true north - this wont work when the vertical pattern will be supported, but we are far from that.
We can still mark the azimuth as required or not (I let Kate decide). If the SAS operator wants to specify the pattern according to true north, he needs to specify the antennaAzimuth to 0 either explicitely (if required) or implictely (if optional)

Reply:
I checked with Yi and he says we should consider antenna azimuth of the ESC sensor to be required. I think this will reduce the logic in SAS/test harness slightly and reduce the chance for silly data entry errors, with minimal (if any?) cost.